packages/app-bridge/src/types/PrivacySettings.ts

```diff
@@ -0,0 +1,6 @@
+/* (c) Copyright Frontify Ltd., all rights reserved. */
+
+export type PrivacySettings = {
+    assetViewerEnabled?: boolean;
+    imageAndVideoDownloadEnabled?: boolean;
```

Comment:
```suggestion
    assetViewerEnabled: boolean;
    assetDownloadEnabled: boolean;
```
`imageAndVideo` => `asset`?
Can you make sure it is defined when you implements the 2 methods? (removed the `?` in the suggestion)